.circleci/config.yml

```diff
@@ -26,8 +26,18 @@ jobs:
             mkdir -p $CIRCLE_ARTIFACTS
             mkdir -p $CIRCLE_TEST_REPORTS
 
+      - restore_cache:
+          name: "Restoring npm cache"
+          key: v1-{{ checksum ".nvmrc" }}-{{ checksum "npm-shrinkwrap.json" }}-npmcache
```

Comment:
this might be a good place to use the [multi-level caching](https://circleci.com/docs/2.0/caching/) circle supports
so
```
keys:
  - v1-npmcache-{{ checksum ".nvmrc" }}-{{ checksum "npm-shrinkwrap.json" }}
  - v1-npmcache-{{ checksum ".nvmrc" }}-
```

Reply:
Nice idea (I think)!

That would:
- Match `.nvmrc`  _and_ `npm-shrinkwrap`
- Fallback on miss to matching just `.nvmrc`
- Cache miss if no match

Seems like a good improvement!